### PR TITLE
Add Shout feature

### PR DESCRIPTION
**Pull request by rajatgoyal715**

### CODE REVIEW

**Comment by therealshabi** [comment]
Can you please write a Javadoc for the classes, methods and variables used, so that the functions becomes self-explanatory. Also, rebase your branch to the dev branch, I want to first merge it to the dev branch, test it and then maybe merge to master after that.

**Review by therealshabi — CHANGES_REQUESTED** [review]
Can you please write a Javadoc for the classes, methods and variables used, so that the functions becomes self-explanatory. Also, rebase your branch to the dev branch, I want to first merge it to the dev branch, test it and then maybe merge to master after that.

**Comment by rajatgoyal715** [comment]
@therealshabi Please review the changes once again and tell me if anything else is needed.

File: aqlite/src/main/java/technolifestyle/com/aqlite/AQLite.kt [hunk 1]
```diff
@@ -35,4 +35,18 @@ class AQLite(private var view: View) {
         view.background = ContextCompat.getDrawable(context, drawableRes)
     }
 
+    fun shout(message: String, type: Type = Type.DEBUG, delimiter: Char = '*') {
+        val shout = Shout(context)
+        shout.shout(message, type, delimiter)
+    }
+
+}
+
```
Comment (therealshabi) [inline aqlite/src/main/java/technolifestyle/com/aqlite/AQLite.kt:50]: Javadoc missing

Comment (therealshabi) [reply aqlite/src/main/java/technolifestyle/com/aqlite/AQLite.kt:50]: Add for both the method and the enum

**Pull request merged**